docs/ossm/ossm2-migration/multi-tenancy/README.md

```diff
@@ -0,0 +1,285 @@
+# OSSM 2 --> 3 Multi-tenancy Migration guide
+
+This guide is for users who are currently running `MultiTenant` OSSM 2.x migrating to OSSM 3.x. You should first read [this document comparing OSSM 2 vs. OSSM 3](../../ossm2-vs-ossm3.md) to familiarize yourself with the concepts between the two versions and the differences in manging `MultiTenant` workloads. Specifically the [Scoping the Mesh section](../../ossm2-vs-ossm3.md#scoping-of-the-mesh-discovery-selectors-and-labels-replace-servicemeshmemberroll-and-servicemeshmember) is important for migrating from OSSM 2 to OSSM 3.
+
+## Migrating OSSM 2 Multi-Tenant to OSSM 3
+
+### Prerequisites
+
+- OSSM2 operator is installed
+- OSSM3 operator is installed
+- `IstioCNI` is installed
+- `istioctl` is installed
+- MultiTenant SMCP instance
+
+### Procedure
```

Comment:
As there's a lot of steps here, I'd consider breaking the procedure up with subheadings... off hand, it looks like:
1. Install OSSM 2.6 and deploy bookinfo
2. Install OSSM 3.0
3. Migrate workloads